src/main/scala/cromwell/backend/model/TaskStatus.scala

```diff
@@ -1,21 +1,24 @@
 package cromwell.backend.model
 
-import cromwell.backend.model.Status.Status
+import cromwell.caching.ExecutionHash
+import wdl4s.values.WdlValue
 
-/**
-  * Represents status of an execution.
-  */
-object Status extends Enumeration {
-  type Status = Value
-  val Created, Running, Succeeded, Failed, Canceled = Value
-}
+import scala.concurrent.Future
 
-/**
-  * Defines a task intermediate status.
-  */
-case class TaskStatus(status: Status) extends ExecutionEvent
 
-/**
-  * Defines a task final status with the resulting data.
-  */
-case class TaskFinalStatus(status: Status, result: ExecutionResult) extends ExecutionEvent
\ No newline at end of file
+sealed trait TaskStatus extends ExecutionEvent
+
+sealed trait IntermediateTaskStatus extends TaskStatus
+final case class CreatedTaskStatus(stdout: String, stderr: String) extends IntermediateTaskStatus
```

Comment:
In an ideal world for me, I think it should be part of the final task result. 
But due to the current context we have, I think it will work.